ciphers/rsa.py

```diff
@@ -0,0 +1,60 @@
+def multiplicative_inverse(a, m) : 
```

Comment:
Please add one or more [__doctests__](https://docs.python.org/3/library/doctest.html) for this function.